governance/program/src/instruction.rs

```diff
@@ -0,0 +1,247 @@
+//! Program instructions
+
+use solana_program::{epoch_schedule::Slot, instruction::Instruction, pubkey::Pubkey};
+
+use crate::state::{GoverningTokenType, Vote, VoteWeight};
+
+/// Instructions supported by the Governance program
+#[derive(Clone)]
+#[allow(clippy::large_enum_variant)]
+pub enum GovernanceInstruction {
+    /// Creates Governance Realm account which aggregates governances for given Governance Mint and optional Council Mint
+    ///
+    /// 0. `[writable]` Governance Realm account. PDA seeds:['governance',name]
+    /// 1. `[]` Governance Token Mint
+    /// 2. `[writable]` Governances Token Holding account
```

Comment:
Please document that account 0 must be the set as the SPL Token owner of this holding account, same with the council token mint

Reply:
Done